3.7.0 release fails to build with bazel due to missing error prone dependency

**What version of protobuf and what language are you using?**
Version: 3.7.0
Language: Java

**What operating system (Linux, Windows, ...) and version?**

Debian based linux.

**What runtime / compiler are you using (e.g., python version or gcc version)**

**What did you do?**
`bazel build //:protobuf_java_util`

**What did you expect to see**
`Build completed successfully`

**What did you see instead?**
```
$ bazel build //:protobuf_java_util
Starting local Bazel server and connecting to it...
INFO: Analysed target //:protobuf_java_util (16 packages loaded, 961 targets configured).
INFO: Found 1 target...
ERROR: /home/apilloud/protobuf-3.7.0/BUILD:632:1: Building libprotobuf_java_util.jar (6 source files) failed (Exit 1)
java/util/src/main/java/com/google/protobuf/util/Timestamps.java:39: error: package com.google.errorprone.annotations does not exist
import com.google.errorprone.annotations.CanIgnoreReturnValue;
                                        ^
java/util/src/main/java/com/google/protobuf/util/Durations.java:45: error: package com.google.errorprone.annotations does not exist
import com.google.errorprone.annotations.CanIgnoreReturnValue;
                                        ^
java/util/src/main/java/com/google/protobuf/util/FieldMaskTree.java:33: error: package com.google.errorprone.annotations does not exist
import com.google.errorprone.annotations.CanIgnoreReturnValue;
                                        ^
java/util/src/main/java/com/google/protobuf/util/FieldMaskUtil.java:39: error: package com.google.errorprone.annotations does not exist
import com.google.errorprone.annotations.CanIgnoreReturnValue;
                                        ^
java/util/src/main/java/com/google/protobuf/util/JsonFormat.java:35: error: package com.google.errorprone.annotations does not exist
import com.google.errorprone.annotations.CanIgnoreReturnValue;
                                        ^
java/util/src/main/java/com/google/protobuf/util/Durations.java:155: error: cannot find symbol
  @CanIgnoreReturnValue
   ^
  symbol:   class CanIgnoreReturnValue
  location: class Durations
java/util/src/main/java/com/google/protobuf/util/Durations.java:168: error: cannot find symbol
  @CanIgnoreReturnValue
   ^
  symbol:   class CanIgnoreReturnValue
  location: class Durations
java/util/src/main/java/com/google/protobuf/util/Durations.java:179: error: cannot find symbol
  @CanIgnoreReturnValue
   ^
  symbol:   class CanIgnoreReturnValue
  location: class Durations
java/util/src/main/java/com/google/protobuf/util/Timestamps.java:166: error: cannot find symbol
  @CanIgnoreReturnValue
   ^
  symbol:   class CanIgnoreReturnValue
  location: class Timestamps
java/util/src/main/java/com/google/protobuf/util/FieldMaskTree.java:99: error: cannot find symbol
  @CanIgnoreReturnValue
   ^
  symbol:   class CanIgnoreReturnValue
  location: class FieldMaskTree
java/util/src/main/java/com/google/protobuf/util/FieldMaskTree.java:129: error: cannot find symbol
  @CanIgnoreReturnValue
   ^
  symbol:   class CanIgnoreReturnValue
  location: class FieldMaskTree
java/util/src/main/java/com/google/protobuf/util/FieldMaskUtil.java:310: error: cannot find symbol
    @CanIgnoreReturnValue
     ^
  symbol:   class CanIgnoreReturnValue
  location: class MergeOptions
java/util/src/main/java/com/google/protobuf/util/FieldMaskUtil.java:323: error: cannot find symbol
    @CanIgnoreReturnValue
     ^
  symbol:   class CanIgnoreReturnValue
  location: class MergeOptions
java/util/src/main/java/com/google/protobuf/util/FieldMaskUtil.java:340: error: cannot find symbol
    @CanIgnoreReturnValue
     ^
  symbol:   class CanIgnoreReturnValue
  location: class MergeOptions
java/util/src/main/java/com/google/protobuf/util/JsonFormat.java:474: error: cannot find symbol
      @CanIgnoreReturnValue
       ^
  symbol:   class CanIgnoreReturnValue
  location: class Builder
java/util/src/main/java/com/google/protobuf/util/JsonFormat.java:487: error: cannot find symbol
      @CanIgnoreReturnValue
       ^
  symbol:   class CanIgnoreReturnValue
  location: class Builder
Target //:protobuf_java_util failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 45.120s, Critical Path: 19.88s
INFO: 183 processes: 182 linux-sandbox, 1 worker.
FAILED: Build did NOT complete successfully
```

**Anything else we should know about your project / environment**

--- a/BUILD
+++ b/BUILD
@@ -639,6 +639,7 @@ java_library(
     visibility = ["//visibility:public"],
     deps = [
         "protobuf_java",
+        "//external:error_prone_annotations",
         "//external:gson",
         "//external:guava",
     ],

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -65,3 +65,13 @@ bind(
     name = "gson",
     actual = "@gson_maven//jar",
 )
+
+maven_jar(
+    name = "error_prone_annotations_maven",
+    artifact = "com.google.errorprone:error_prone_annotations:2.3.2",
+)
+
+bind(
+    name = "error_prone_annotations",
+    actual = "@error_prone_annotations_maven//jar",
+)

